tests/src/rules/prefer-default-export.js

```diff
@@ -129,4 +130,62 @@ ruleTester.run('prefer-default-export', rule, {
       }],
     }),
   ],
-})
+});
+
+context('Typescript', function() {
+  // Typescript
+  const parsers = [require.resolve('typescript-eslint-parser'), require.resolve('babel-eslint')];
```

Comment:
oops, i think typescript-eslint-parser can only work in eslint < 6

Reply:
that makes sense, i updated this with another conditional that checks for <v6